TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/full_tests.py

```diff
@@ -131,9 +150,10 @@ def test_e2e_0001_default(self):
         replayer.start()
         wait_for_running_replayer(replayer=replayer)
 
-        expected_docs[index_name] = {"count": 3}
-        check_doc_counts_match(cluster=source_cluster, expected_index_details=expected_docs,
+        expected_source_docs[index_name] = {"count": 3}
+        # TODO Replayer transformation needed to only have docs in the transformed index
```

Comment:
Can you please make a follow-up Jira for this work and link it here?

Reply:
Here: https://opensearch.atlassian.net/browse/MIGRATIONS-2277